.travis.yml

```diff
@@ -3,7 +3,7 @@ services:
 
 language: go
 go:
-  - 1.10.x
+  - 1.10.8
```

Comment:
1.10.x is technically 1.10.8, not sure if this is needed

Reply:
would suggest to add a comment that tests are stuck with 1.11.x and later here instead

Reply:
The tests for the router controller does not work with Go 1.11, i suspect it being due to the old vendored goBGP server